scapy/contrib/lldp.py

```diff
@@ -232,7 +211,7 @@ def _tlv_multiplicities_check(tlv_type_count):
 
         # * : 0..n, 1 : one and only one.
         standard_multiplicities = {
-            LLDPDUEndOfLLDPDU.__name__: 1,
+            LLDPDUEndOfLLDPDU.__name__: '*',
```

Comment:
If I understand this well, this would allow multiple LLDPDUEndOfLLDPDU. Is this desirable?

Reply:
Yes frame can have multiple `End Of LLDPDU TLV`
as per `8.5.1 End Of LLDPDU TLV`:
```
The End Of LLDPDU TLV is a 2-octet, all-zero TLV that is used to mark the end of the TLV sequence in LLDPDUs.
...
NOTE—Some IEEE 802 MACs require the data field in a frame to contain a minimum number of octets. ... . Since pad octets are unspecified, an End Of LLDPDU TLV can be used to prevent non-zero pad octets from being interpreted by the receiving LLDP agent as another TLV.
```

The only part I do not know how to do is to stop frame parsing after seeing first `End Of LLDPDU TLV`